core/migrate/data.go

```diff
@@ -42,4 +42,10 @@ var migrations = []migration{
 			DROP COLUMN index;
 		ALTER TABLE account_utxos ADD PRIMARY KEY (output_id);
 	`},
+	{Name: `2017-03-02.0.core.add-output-source-info.sql`, SQL: `
+		ALTER TABLE account_utxos
+			ADD COLUMN source_id bytea NOT NULL,
+			ADD COLUMN source_pos bigint NOT NULL,
+			ADD COLUMN ref_data_hash bytea NOT NULL;
```

Comment:
We expect customers to completely drop the database and recreate it before attempting to run a 1.1.1 binary right? I think that's a fine assumption when dealing with a blockchain reset.

Once landed, we should double check that the migration has the same name and hash on the 1.1.1 branch and `main` so that Cores updating from 1.1.1 to 1.2.0 do not try to run this migration again.